### ISSUE TITLE
Header offsetting from the current position

### ISSUE THREAD

**Issue by ishantvivek** [tonytomov/jqGrid#1047]
Hello team,

I am having one issue that when we make columns: `sortable:true` and we have lots of columns with a scroll bar. On dragging, the selected column moves disproportionate.

Here is the screenshot:
![image](https://github.com/tonytomov/jqGrid/assets/24982790/55833eb8-2fa2-4cbd-9515-d7d7f8b9827d)

jqGrid 4.13.4

**Comment by ishantvivek** [comment]
Note: I debugged the issue a little bit, and I found when we start dragging, it appends `left, right and position` to the element, however the value for `left` it is calculating is wrong. This happens only when we have scrolled the grid to right.  If scrollbar is at 0, this issue doesn't happen.

**Comment by ishantvivek** [comment]
The latest version is even a big mess:

![image](https://github.com/tonytomov/jqGrid/assets/24982790/050f9326-5419-45d2-8632-1b620f55ebe5)

**Comment by parallels999** [comment]
> The latest version is even a big mess:

No, you're just too out of date, and you're probably affected by a lot of breaking changes.
However, the version you are using is no longer supported.

**Comment by tonytomov** [comment]
The problem you have (from my expierence) is related with inconsistency from colModel and source data. 
Mostly this is caused with repeatdata : true in jsonReader.

Check your data and colmodel if they are in sync.
